yb-voyager/cmd/cutoverToTargetCmd.go

```diff
@@ -89,12 +89,11 @@ func init() {
 	registerConfigFileFlag(cutoverToTargetCmd)
 	BoolVar(cutoverToTargetCmd.Flags(), &prepareForFallBack, "prepare-for-fall-back", false,
 		"prepare for fallback by streaming changes from target DB back to source DB. Not applicable for fall-forward workflow.")
-	BoolVar(cutoverToTargetCmd.Flags(), &useYBgRPCConnector, "use-yb-grpc-connector", true,
-		`YugabyteDB provides two types of CDC (Change Data Capture) connectors:
+	BoolVar(cutoverToTargetCmd.Flags(), &useYBgRPCConnector, "use-yb-grpc-connector", false,
```

Comment:
We don't want to change the default, do we? 
how is this set to be a mandatory flag, btw?

Reply:
Ah, I see. Pls add a comment above to explain this behavior, because it's a bit confusing when looking at the code (default=false)